src/alire/alire-environment.adb

```diff
@@ -269,6 +315,8 @@ package body Alire.Environment is
    procedure Export (This : Context) is
    begin
       for Var of This.Compile loop
+         Trace.Debug ("Exporting environment: "
```

Comment:
There is already a debug message in `Setenv`.

Reply:
Good catch